package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.11",
     "@react-native-community/blur": "^4.3.0",
+    "@react-native-community/checkbox": "^0.5.13",
```

Comment:
Please use toggles instead of checkboxes whenever you need them. For examples on usage, see `screens/Settings/stacks/PlaybackReporting.tsx`.